WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -75,15 +75,29 @@ private extension CardReaderSettingsConnectedViewController {
     ///
     func configure(_ cell: UITableViewCell, for row: Row, at indexPath: IndexPath) {
         switch cell {
-        case let cell as TitleTableViewCell where row == .temporaryHeader:
-            configureHeader(cell: cell)
+        case let cell as ConnectedReaderTableViewCell where row == .connectedReader:
+            configureConnectedReader(cell: cell)
+        case let cell as ButtonTableViewCell where row == .disconnectButton:
+            configureButton(cell: cell)
         default:
             fatalError()
         }
     }
 
-    private func configureHeader(cell: TitleTableViewCell) {
-        cell.titleLabel?.text = Localization.temporaryHeader
+    private func configureConnectedReader(cell: ConnectedReaderTableViewCell) {
+        guard let serialNumber = viewModel?.connectedReaderSerialNumber,
+              let batteryLevel = viewModel?.connectedReaderBatteryLevel else {
+            return
+        }
+        cell.serialNumberLabel?.text = serialNumber
+        cell.batteryLevelLabel?.text = batteryLevel
+        cell.selectionStyle = .none
+    }
+
+    private func configureButton(cell: ButtonTableViewCell) {
+        cell.configure(title: Localization.buttonTitle) {
+            // TODO: Connect in 4057
```

Comment:
PR #4140 implements support for disconnection in the CardPresentPaymentStore so this should be an easy win 😉